test/test_cpp_api_parity.py

```diff
@@ -225,13 +252,26 @@ def _test_torch_nn_module_ctor_args(self, module_name):
         cpp_module_option = 'torch::nn::{}Options{}'.format(module_name, cpp_default_constructor_args_str)
         init_kwargs = init_arg_spec.args[-len(init_kwargs_defaults):]
         for arg_name, python_default_value in zip(init_kwargs, init_kwargs_defaults):
-            cpp_module_option += '.{}({})'.format(arg_name, self._python_arg_to_cpp_arg(python_default_value).value)
+            # NOTE: If a Python module constructor arg's default value is None, we don't test its corresponding
+            # options arg in C++ module (because the way to set the C++ options arg to an empty value is to not
+            # specify it, which means we can't test that the options arg exists).
+            # Instead, we test that all options args exist by calling their accessors after constructing the
+            # C++ module with the options.
+            if python_default_value is not None:
+                cpp_module_option += '.{}({})'.format(arg_name, self._python_arg_to_cpp_arg(python_default_value).value)
+
+        extra_stmts = []
```

Comment:
nit: If you wanted to save three lines of code, this is just (ignore the crazy indentation):
```
extra_stmts = [TORCH_NN_MODULE_TEST_OPTIONS_ARG.substitute(options_arg_name=arg_name)
                         for arg_name in  python_default_constructor_arg_names + init_kwargs]